content.md

```diff
@@ -2,8 +2,12 @@
-- `git fetch` - get changes from a remote repo into your tracking branch
-- `git pull` - will get changes from a remote branch into your tracking branch and merge them into a local branch
+- `git push` - sends changes from a local branch to a remote repository
+- `git fetch` - adds changes from a remote repository into your tracking branch
+- `git pull` - adds changes from a remote branch into your tracking branch and merges these changes into your local branch
 
-Often `git push` and `git pull` are described as equivalent. This isn't entirely correct, since under the hood `git pull` does two things. `git push` takes our current branch, and checks to see whether or not there is a tracking branch for a remote repository connected to it. If so, our changes are taken from our branch and pushed to the remote branch. This is how code is shared with a remote repository, you can think of it as "make the remote branch resemble my local branch". This will fail if the remote branch has diverged from your local branch: if not all the commits in the remote branch are in your local branch. When this happens, your local branch needs to be synchronized with the remote branch with git pull or git fetch and git merge.`git fetch` again takes our current branch, and checks to see if there is a tracking branch. If so, it looks for changes in the remote branch, and pulls them into the tracking branch. It does not change your local branch. To do that, you'll need to do `git merge origin/master` (for the "master" branch) to merge those changes into your branch - probably also called "master".`git pull` simply does a `git fetch` followed immediately by `git merge`. This is often what we desire to do, but some people prefer to use git fetch followed by git merge to make sure they understand the changes they are merging into their branch before the merge happens.
\ No newline at end of file
+`git push` first identifies if your tracking branch has a connection to a remote repository. If a connection exists, `git push` adds changes from your tracking branch to the remote branch, allowing you to share code with the remote repository and to make the remote branch remsemble your local branch. If the remote branch has diverged from your local branch, the push will fail. When this occurs, synchronize your local branch to the remote branch by using `git pull` or by executing `git fetch` followed by `git merge`. 
+
+`git fetch` first identifies if your tracking branch has a connection to your local branch. If a connection exists, `git fetch` adds changes from the remote branch to your tracking branch without altering data in your local branch. To merge changes from the remote branch into your local branch, execute `git merge origin/[branch name]`.
+
+`git pull` executes `git fetch` immediately followed by `git merge`. While this is often the desired result, many developers prefer to use `git fetch` followed by `git merge` to understand all code changes prior to the merge.
```

Comment:
Converts passive voice to active voice; removes unnecessary simplification ("simply"); other edits for clarity and flow.